server/@types/ui/index.d.ts

```diff
@@ -50,7 +50,17 @@ type HasHtmlString = {
   html: string
 }
 
-type TagColour = 'blue' | 'green' | 'grey' | 'orange' | 'pink' | 'purple' | 'red' | 'turquoise' | 'yellow'
+type TagColour =
+  | 'blue'
+  | 'green'
+  | 'grey'
+  | 'light-blue'
+  | 'orange'
+  | 'pink'
+  | 'purple'
+  | 'red'
+  | 'turquoise'
+  | 'yellow'
```

Comment:
I think you can get this from backend? 🤔

Reply:
We do, but more for type safety when testing and we still use `TagColour` for the audience tags.